### ISSUE TITLE
Enable non-Finnish speakers to change language settings from their screen readers

### ISSUE THREAD

**Issue by kmhentto** [City-of-Helsinki/kerrokantasi-ui#777]
Visually impaired people cannot change language from Finnish to English or Swedish from their screen readers (if the language of the screen reader is other than Finnish).  This need to be fixed, description in Finnish below.

Sivuston kieleksi voi valita suomen, ruotsin tai englannin. Kieli valitaan valikosta, joka on nimetty avustavalle teknologialle ”Valitse sivuston kieli” -nimiöinnillä. Nimiöinti vaihtuu kulloinkin valittuna olevan kielen mukaiseksi noudattaen samaa nimiöintikaavaa. Näkövammainen vieraskielinen ruudunlukuohjelman käyttäjä ei pysty nykytoteutuksessa vaihtamaan sivuston kieltä esimerkiksi suo-mesta englanniksi. Kielivalikko on mahdoton löytää ruudunlukuohjelmalla, jos käyttäjä ei osaa suomea.
Lisäksi kielten valintaelementeistä puuttuvat kieltä vastaavat lang-kielimääritykset, joten ruudunlukuohjelma lausuu kielet väärän kielen lausuntatapaa käyttäen.
**Korjausehdotus**
Kielivalinta tulee toteuttaa suorina linkkeinä, jotka nimetään Suomi, Svenska, English, ilman erillistä kielivalikkoa. Lisäksi kielilinkeille tu-lee asettaa kieltä vastaavat lang-kielimääritykset.

**Comment by JoonasNislin** [comment]
Merged to master
